lib/net/parser.js

```diff
@@ -140,11 +140,6 @@ class Parser extends EventEmitter {
     let i = 0;
     for (; data[i + 4] !== 0 && i < 12; i++);
 
-    if (i === 12) {
```

Comment:
Could you explain why this is removed?

Reply:
I looked into this previously and it's because of the `getcfheaders` message, of which is not `null` terminated and uses the maximum space available.

Reply:
I think this error was originally added based on https://en.bitcoin.it/wiki/Protocol_documentation#Message_structure which defines the command as:\
> "ASCII string identifying the packet content, NULL padded (non-NULL padding results in packet rejected)". 

However, there may be an exception to "non-NULL padding results in packet rejection", if the message uses the entire 12 bytes.